packages/twenty-front/src/modules/object-record/record-field/components/FormFieldInput.tsx

```diff
@@ -57,8 +60,9 @@ export const FormFieldInput = ({
       label={field.label}
       defaultValue={defaultValue as string | undefined}
       onPersist={onPersist}
-      field={field}
       VariablePicker={VariablePicker}
+      options={field.metadata.options}
+      clearLabel={field.label}
```

Comment:
Dissociating the field definition from the input is excellent. I'm unsure if it will "scale" well as the other fields make extensive use of the `useNumberField`-like functions and get access to the field definition that way.

Reply:
let's see. If it becomes too complex we will still be able to come back to full field!